aws/structure.go

```diff
@@ -160,6 +160,32 @@ func expandEcsVolumes(configured []interface{}) ([]*ecs.Volume, error) {
 			if v, ok := config["root_directory"].(string); ok && v != "" {
 				l.EfsVolumeConfiguration.RootDirectory = aws.String(v)
 			}
+
+			if v, ok := config["transit_encryption"].(bool); ok {
+				if v {
+					l.EfsVolumeConfiguration.TransitEncryption = aws.String("ENABLED")
+				} else if !v {
+					l.EfsVolumeConfiguration.TransitEncryption = aws.String("DISABLED")
+				}
+			}
+
+			if v, ok := config["transit_encryption_port"].(int64); ok && v != 0 {
+				l.EfsVolumeConfiguration.TransitEncryptionPort = aws.Int64(v)
+			}
+			if v, ok := config["authorization_config"].(map[string]interface{}); ok && len(v) > 0 {
```

Comment:
Following the patter, maybe it would be best to have something like:
```golang
authConfig, ok := efsconfig["authorization_config"].([]interface{})
if ok && len(authConfig) > 0 {
	authconfig := authConfig[0].(map[string]interface{})
	l.EfsVolumeConfiguration.AuthorizationConfig = &ecs.EFSAuthorizationConfig{}

	if v, ok := authconfig["access_point_id"].(string); ok && v != "" {
		l.EfsVolumeConfiguration.AuthorizationConfig.AccessPointId = aws.String(v)
	}

	if v, ok := authconfig["iam"].(string); ok && v != "" {
		l.EfsVolumeConfiguration.AuthorizationConfig.Iam = aws.String(v)
	}
}
```
And rename `config` to `efsconfig` or similar. 👍

Reply:
Yes. Your implementation is correct. The rename of `config` was a suggestion as it would increase readability. Works for me. Great work.